debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -3044,6 +3044,31 @@ public void customYBStructureShouldBePresentInSnapshotRecords() throws Exception
         assertEquals(expectedPKValues, actualPKValues);
     }
 
+    @Test
+    public void shouldOnlyMarkColumnsAsRequiredWithReplicaIdentityIsChange() throws Exception {
```

Comment:
Could we also have a test for Replica Identity DEFAULT

Reply:
Added tests.